src/components/transaction/TransactionTable/index.tsx

```diff
@@ -0,0 +1,294 @@
+      }
+    }
+    tr > td {
+      display: flex;
+      flex: 1;
+      width: 100%;
+      justify-content: space-between;
+      margin: 0;
+      margin-bottom: 18px;
+      min-height: 32px;
+      span.span-inside-tooltip {
+        align-items: flex-end;
+        flex-direction: column;
+        img {
+          margin-left: 0;
+        }
+      }
+    }
+    .header-value {
+      flex-wrap: wrap;
+      text-align: end;
+    }
+    .span-copybtn-wrap {
+      display: flex;
+      flex-wrap: nowrap;
+      span {
+        display: flex;
+        align-items: center;
+      }
+      .copy-text {
+        margin-left: 5px;
+      }
+    }
+  }
+  overflow: auto;
+`
+
+const HeaderTitle = styled.span`
+  display: none;
+  ${media.mediumDown} {
+    font-weight: 600;
+    align-items: center;
+    display: flex;
+    margin-right: 3rem;
+    svg {
+      margin-left: 5px;
+    }
+  }
+`
+const HeaderValue = styled.span`
+  ${media.mediumDown} {
+    flex-wrap: wrap;
+    text-align: end;
+  }
+`
+
+function getLimitPrice(transaction: Order, isPriceInverted: boolean): string {
+  if (!transaction.buyToken || !transaction.sellToken) return '-'
+
+  const calculatedPrice = getOrderLimitPrice({
+    buyAmount: transaction.buyAmount,
+    sellAmount: transaction.sellAmount,
+    buyTokenDecimals: transaction.buyToken.decimals,
+    sellTokenDecimals: transaction.sellToken.decimals,
+    inverted: isPriceInverted,
+  })
+
+  return formatCalculatedPriceToDisplay(calculatedPrice, transaction.buyToken, transaction.sellToken, isPriceInverted)
+}
+
+const tooltip = {
+  tradeID: 'A unique identifier ID for this trade.',
+}
+
+export type Props = StyledUserDetailsTableProps & {
+  transactions: Order[]
+  trades: Trade[]
+}
+
+interface RowProps {
+  transaction: Order
+  isPriceInverted: boolean
+  invertLimitPrice: () => void
+}
+
+const RowTransaction: React.FC<RowProps> = ({ transaction, isPriceInverted, invertLimitPrice }) => {
+  const {
+    buyToken,
+    buyAmount,
+    expirationDate,
+    partiallyFilled = false,
+    sellToken,
+    sellAmount,
+    kind,
+    txHash,
+    shortId,
+  } = transaction
+  const network = useNetworkId()
+  const buyTokenSymbol = buyToken ? safeTokenName(buyToken) : ''
+  const sellTokenSymbol = sellToken ? safeTokenName(sellToken) : ''
+  const sellFormattedAmount = formattedAmount(sellToken, sellAmount)
+  const buyFormattedAmount = formattedAmount(buyToken, buyAmount)
+  const renderSpinnerWhenNoValue = (textValue: string): JSX.Element | void => {
+    if (textValue === '-') return <FontAwesomeIcon icon={faSpinner} spin size="1x" />
+  }
+  const limitPriceSettled = getLimitPrice(transaction, isPriceInverted)
+
+  return (
+    <tr key={txHash}>
+      <td>
+        <HeaderTitle>
+          Order ID <HelpTooltip tooltip={tooltip} />
+        </HeaderTitle>
+        <HeaderValue>
+          <RowWithCopyButton
+            className="span-copybtn-wrap"
+            textToCopy={shortId}
```

Comment:
The copy hash problem that @elena-zh mention [`here`](https://github.com/gnosis/gp-ui/pull/897#issuecomment-995895355) is because that you are copying `shortId` instead of complete txHash

Reply:
yes, i saw that, thanks!